script/1_0_2current_en_script/Arcueid/Day 4/04_00_ARC04_6_1.txt

```diff
@@ -42,9 +42,9 @@ In that case, lend me your phone. I don't want to use mine for this."
 <Page13>
 "―――,# Ha."
 A breath of relief escaped my lips at the unexpected offer.
-Just as the girl says, it seems that I had sunk into a malaise without noticing. [[FIXME]]//Orig been experiencing a malfunction
+Just as the girl says, it seems that I had sunk into a malaise without noticing.//Orig been experiencing a malfunction, to be rechecked
```

Comment:
Back to this line again... I still think malaise is a pretty appropriate medical term here, but am open to alternatives that read as a medical condition.
> 少女の言う通り、気づいていないだけで機能不全に陥っていたらしい。